internal/vppinit/vppinit.go

```diff
@@ -157,6 +166,53 @@ func createAfPacket(ctx context.Context, vppConn api.Connection, link netlink.Li
 	return afPacketCreateRsp.SwIfIndex, nil
 }
 
+func addArp(ctx context.Context, vppConn api.Connection, swIfIndex interface_types.InterfaceIndex) error {
+	f, err := os.OpenFile("/proc/net/arp", os.O_RDONLY, 0600)
+	if err != nil {
+		return err
+	}
+	defer func() { _ = f.Close() }()
+
+	reader := bufio.NewReader(f)
+	for l := 0; ; l++ {
+		now := time.Now()
+		line, err := reader.ReadString('\n')
+		if err != nil {
+			break
+		}
+		// Skip the first line with headers
+		if l == 0 {
+			continue
+		}
+		parts := strings.Fields(strings.TrimSpace(line))
```

Comment:
```suggestion
		fields := strings.Fields(strings.TrimSpace(line))
```

Also we could declare a consts that can remove magic indexes.

For example:

```go
const (
	ipAddr int = iota
	hwType
	flags
	hwAddr
	mask
	device
)
```

Reply:
Lets follow https://github.com/networkservicemesh/cmd-forwarder-vpp/pull/186#discussion_r631096359